src/openforms/registrations/contrib/zgw_apis/migrations/0007_move_singleton_data.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 3.2.19 on 2023-06-08 09:37
+
+from django.db import migrations
+
+
+def move_singleton_data_to_zgw_api_set_model(apps, _):
+    ZgwConfig = apps.get_model("zgw_apis", "ZgwConfig")
+    ZGWApiGroupConfig = apps.get_model("zgw_apis", "ZGWApiGroupConfig")
+
+    solo_model = ZgwConfig.objects.first()
+    if not solo_model:
+        return
+
+    new_config_set = ZGWApiGroupConfig(
+        name="ZGW API",
+        zrc_service=solo_model.zrc_service,
+        drc_service=solo_model.drc_service,
+        ztc_service=solo_model.ztc_service,
+        zaaktype=solo_model.zaaktype,
+        informatieobjecttype=solo_model.informatieobjecttype,
+        organisatie_rsin=solo_model.organisatie_rsin,
+        zaak_vertrouwelijkheidaanduiding=solo_model.zaak_vertrouwelijkheidaanduiding,
+        doc_vertrouwelijkheidaanduiding=solo_model.doc_vertrouwelijkheidaanduiding,
+        auteur=solo_model.auteur,
+    )
+    new_config_set.save()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("zgw_apis", "0006_zgwapigroupconfig"),
+    ]
+
+    operations = [
+        migrations.RunPython(
+            move_singleton_data_to_zgw_api_set_model, migrations.RunPython.noop
```

Comment:
Now migrating forwards and then backwards again leads to data loss. Can you implement the reverse operation where you grab the attached group config put that config back in the singleton?